Pipfile

```diff
@@ -6,9 +6,9 @@ verify_ssl = true
 [dev-packages]
 mock = "*"
 requests-mock = "*"
-jinja2 = "*"
```

Comment:
How come jinja2 isn't needed anymore?

Reply:
jinja2 is used by the target data inconsistencies report which is in /ci_scripts and is not needed by the actual mbed-target code